resources/public/locales/en/default.json

```diff
@@ -71,7 +71,18 @@
         "WelcomeDescription1":"Use this app to verify the SMART Health Cards of your customers or employees at your business.",
         "WelcomeDescription2":"This app does not save or store your digital credentials.",
         "PleaseNote":"Please Note: ",
-        "GetStarted":"Get Started"
+        "GetStarted":"Get Started",
```

Comment:
Add all the new entry to fr/default.json ja/default.json ( just cut and paste English is fine for now )